### PR TITLE
CommitmentTypeIndication check disabled

### PR DESCRIPTION
**Pull request by vt-hle**
Disabled check for CommitmentTypeIndication as it is currently failing for all users.

### CODE REVIEW

File: dss-demo-webapp/src/main/resources/IcelandPolicy.xml [hunk 1]
```diff
@@ -309,9 +309,9 @@
 			<EllipticCurveKeySize Level="WARN" />
 <!--		<ContentType Level="FAIL" value="1.2.840.113549.1.7.1" /> -->
 <!--		<ContentHints Level="FAIL" value="*" /> -->
-			<CommitmentTypeIndication Level="FAIL">
+<!--		<CommitmentTypeIndication Level="FAIL">
```
Comment (brynjarorng) [inline dss-demo-webapp/src/main/resources/IcelandPolicy.xml:312]:
```suggestion
<!--		<CommitmentTypeIndication Level="FAIL"> -->
```

File: dss-demo-webapp/src/main/resources/IcelandPolicy.xml [hunk 2]
```diff
@@ -309,9 +309,9 @@
...
 				<Id>1.2.840.113549.1.9.16.6.5</Id>
-			</CommitmentTypeIndication>
+			</CommitmentTypeIndication> -->
```
Comment (brynjarorng) [inline dss-demo-webapp/src/main/resources/IcelandPolicy.xml:314]:
```suggestion
<!--		</CommitmentTypeIndication> -->
```

File: dss-demo-webapp/src/main/resources/IcelandPolicy.xml [hunk 3]
```diff
@@ -309,9 +309,9 @@
 			<EllipticCurveKeySize Level="WARN" />
 <!--		<ContentType Level="FAIL" value="1.2.840.113549.1.7.1" /> -->
 <!--		<ContentHints Level="FAIL" value="*" /> -->
-			<CommitmentTypeIndication Level="FAIL">
+<!--		<CommitmentTypeIndication Level="FAIL">
 				<Id>1.2.840.113549.1.9.16.6.5</Id>
```
Comment (brynjarorng) [inline dss-demo-webapp/src/main/resources/IcelandPolicy.xml:313]:
```suggestion
<!--				<Id>1.2.840.113549.1.9.16.6.5</Id> -->
```

**Review by brynjarorng — CHANGES_REQUESTED** [review]
Added suggestions related to indentation and formatting

File: dss-demo-webapp/src/main/resources/IcelandPolicy.xml (hunk 1 above)
Comment (vt-hle) [reply dss-demo-webapp/src/main/resources/IcelandPolicy.xml]: Fixed

File: dss-demo-webapp/src/main/resources/IcelandPolicy.xml (hunk 2 above)
Comment (vt-hle) [reply dss-demo-webapp/src/main/resources/IcelandPolicy.xml]: Fixed

**Review by brynjarorng — APPROVED** [review]
LGTM

**Pull request merged**